latest.py

```diff
@@ -8,22 +7,25 @@ def main():
     fork_motor_group.spin_to_position(1800, DEGREES, wait=False)
     goto( -920,  920, 0)
     goto( -920,-1450, 0)
-    pick_up()
-    goto(  750,-1200, 0)
-    set_down()
-    goto( 1000,-1300, 1)
+    goto(  750,-1400, 0) # blue in right zone
+    goto(  500,-1150, 1)
+#   goto(    0, -950, 0) # m = - 2/5
+    goto( -600, -710, 0) # yellow in left zone
+    goto(  400, -250, 1)
+    goto(    0,    0, 0) # m = - 5/8
+    goto( -600,  200, 0) # yellow in left zone
+    goto(  400,  400, 1)
+    goto(    0,  950, 0)
+    goto( -600,  950, 0) # yellow in left zone
+    goto(  920,  950, 1)
     goto(  920, 1500, 0)
-    pick_up()
-    goto( -600, 1320, 0)
-    set_down()
```

Comment:
One more `pick_up()` and `set_down()` pair removed.

Reply:
same as above (line 11 to 14)

Reply:
same as above (line 11 to 14)